app/helpers/patient_helper.rb

```diff
@@ -197,4 +197,62 @@ def self.languages(language)
     }
     languages[language&.downcase&.to_sym].present? ? languages[language&.downcase&.to_sym] : nil
   end
+
+  # Calculated symptom onset date is based on latest symptomatic assessment.
+  def calculated_symptom_onset(patient)
+    patient.assessments.where(symptomatic: true).minimum(:created_at)&.to_date
+  end
+
+  def self.monitoring_fields
+    %i[
+      monitoring
+      monitoring_reason
+      monitoring_plan
+      exposure_risk_assessment
+      public_health_action
+      isolation
+      pause_notifications
+      symptom_onset
+      case_status
+      assigned_user
+      last_date_of_exposure
+      continuous_exposure
+      user_defined_symptom_onset
+      extended_isolation
+      jurisdiction_id
+    ]
+  end
+
+  def self.info_fields
+    %i[
+      isolation
```

Comment:
Should `isolation` be in `monitoring_fields` instead?

Reply:
Currently it is in both arrays intentionally, because `isolation` can be set directly by editing the monitoree (which is why it is in the `info_fields` array), or it can be set indirectly via a `case_status` change from the "Monitoring Actions" section (which is why it is in the `monitoring_fields` array).